standard/structs.md

```diff
@@ -502,6 +504,50 @@ Automatically implemented properties ([§15.7.4](classes.md#1574-automatically-i
 
 > *Note*: This access restriction means that constructors in structs containing automatically implemented properties often need an explicit constructor initializer where they would not otherwise need one, to satisfy the requirement of all fields being definitely assigned before any function member is invoked or the constructor returns. *end note*
 
+A *property_declaration* ([§15.7.1](classes.md#1571-general)) for an instance property in a *struct_declaration* may contain the *property_modifier* `readonly`. However, a static property shall not contain that modifier.
+
+It is a compile-time error to attempt to modify the state of an instance struct variable via a readonly property declared in that struct.
+
+It is a compile-time error for an automatically implemented property having a `readonly` modifier, to also have a `set` accessor.
+
+It is a compile-time error for an automatically implemented property in a `readonly` struct to have a `set` accessor.
+
+An automatically implemented property declared inside a `readonly` struct need not have a `readonly` modifier, as its `get` accessor is implicitly assumed to be readonly.
```

Comment:
Roslyn also treats auto property getter of non-readonly structs as implicitly readonly. Should this be intended in spec?